solutions/cses-1076.mdx

```diff
@@ -87,3 +82,58 @@ int main() {
 	cout << endl;
 }
 ```
+
+## Solution 2: Fenwick Tree
+
+You can use a [Fenwick tree](/gold/PURS#binary-indexed-tree) to simulate an order statistic tree/indexed set.
+
+The Fenwick array can be treated as a frequency array. For example, if `5` is inserted into the window, `Fenwick_array[5]+=1`. This enables us to binary search for an integer $x$ such that 
```

Comment:
Why do you use code for "5"? Also, add spacing to `Fenwick_array[5]+=1` (and maybe rename the array)